---
 Makefile |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

Index: pufferfish/Makefile
===================================================================
--- pufferfish.orig/Makefile	2015-02-11 13:15:29.495897797 +0100
+++ pufferfish/Makefile	2015-02-11 13:24:10.565818636 +0100
@@ -2,7 +2,11 @@ CC        = cc
 CFLAGS    = -g -Wall -pedantic -O3 -march=native
 LIBS      = -lcrypto
 
-all:  pfcrypt-ref pfkdf-ref
+#all:  pfcrypt-ref pfkdf-ref
+all: pufferfish.a
+
+pufferfish.a: pufferfish-ref.o itoa64.o api-ref.o
+	ar rcs $@ reference/pufferfish-ref.o common/itoa64.o common/api-ref.o
 
 pfcrypt-ref: pfcrypt-ref.o pufferfish-ref.o itoa64.o api-ref.o
 	$(CC) -o pfcrypt-ref examples/pfcrypt-ref.o reference/pufferfish-ref.o common/itoa64.o common/api-ref.o $(LIBS)
@@ -26,7 +30,7 @@ itoa64.o:
 	$(CC) -c $(CFLAGS) -o common/itoa64.o common/itoa64.c
 
 clean:
-	rm -f pfcrypt-ref pfkdf-ref examples/*.o common/*.o reference/*.o optimized/*.o
+	rm -f pfcrypt-ref pfkdf-ref examples/*.o common/*.o reference/*.o optimized/*.o *.a
 
 distclean: clean
 
