---
 Makefile |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

Index: parallel/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ parallel/Makefile	2015-04-28 12:43:26.857780816 +0200
@@ -0,0 +1,16 @@
+CC       = g++
+CPPFLAGS   = -c -Wall -O2
+
+TARGETS=parallel.a
+
+SOURCES=parallel.cpp sha512.cpp
+
+default: $(TARGETS)
+
+parallel.a: $(SOURCES:.cpp=.o)
+	ar rcs $@ $^
+
+clean:
+	rm -f *.o $(TARGETS)
+
+.PHONY: clean
