---
 Makefile |   12 ++++++++++++
 1 file changed, 12 insertions(+)

Index: pomelo/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ pomelo/Makefile	2015-02-11 13:12:41.199108051 +0100
@@ -0,0 +1,12 @@
+CFLAGS=-O2 -Wall
+CC=gcc
+
+OBJ=pomelo_sse2.o
+
+pomelo.a: $(OBJ)
+	ar rcs $@ $^
+
+clean:
+	rm -f *.o *~ core *.a
+
+.PHONY: clean
