---
 Lyra2.c     |    2 ++
 sse/Lyra2.c |    3 +++
 2 files changed, 5 insertions(+)

Index: lyra2/Lyra2.c
===================================================================
--- lyra2.orig/Lyra2.c	2014-04-08 09:07:06.000000000 +0200
+++ lyra2/Lyra2.c	2014-09-20 16:55:48.468095223 +0200
@@ -81,6 +81,8 @@ int LYRA2(unsigned char *K, int kLen, co
     int i; //auxiliary iteration counter
     //==========================================================================/
 
+    if (nRows < 3)
+      return -1;
 
     //========== Initializing the Memory Matrix and pointers to it =============//
     //Allocates enough space for the whole memory matrix
Index: lyra2/sse/Lyra2.c
===================================================================
--- lyra2.orig/sse/Lyra2.c	2014-04-08 09:07:06.000000000 +0200
+++ lyra2/sse/Lyra2.c	2014-09-20 17:01:38.320149085 +0200
@@ -90,6 +90,9 @@ int LYRA2(unsigned char *K, int kLen, co
         return -1;
     }
 
+    if (nRows < 3)
+      return -1;
+
     //============================= Basic variables ============================//
     int row = 2; //index of row to be processed
     int prev = 1; //index of prev (last row ever computed/modified)
