---
 Lyra2.c     |   10 +++++++++-
 sse/Lyra2.c |   12 ++++++++++--
 2 files changed, 19 insertions(+), 3 deletions(-)

Index: lyra2/Lyra2.c
===================================================================
--- lyra2.orig/Lyra2.c	2015-04-01 19:08:51.409808656 +0200
+++ lyra2/Lyra2.c	2015-04-01 19:08:51.405808640 +0200
@@ -85,7 +85,10 @@ int LYRA2(void *K, unsigned int kLen, co
 
     uint64_t i;                 //auxiliary iteration counter
     //==========================================================================/
-    
+    if (nRows < 3)
+      return -1;
+    if (timeCost < 1)
+      return -1;
     //========== Initializing the Memory Matrix and pointers to it =============//
     //Tries to allocate enough space for the whole memory matrix
     i = (uint64_t) ((uint64_t) nRows * (uint64_t) ROW_LEN_BYTES);
@@ -291,6 +294,11 @@ int LYRA2(void *K, unsigned int kLen, co
         return -1;
     }
 
+    if (nRows < 3)
+      return -1;
+    if (timeCost < 1)
+      return -1;
+
 #if _OPENMP <= 201107  //OpenMP 3.X or less 
     #pragma omp parallel num_threads(nPARALLEL) default(none) /*private(pwd)*/ shared(memMatrix,  pKeys, pwd, pwdlen, salt, saltlen, nRows, nCols, kLen, timeCost)
 #endif // _OPENMP
Index: lyra2/sse/Lyra2.c
===================================================================
--- lyra2.orig/sse/Lyra2.c	2015-04-01 19:08:51.409808656 +0200
+++ lyra2/sse/Lyra2.c	2015-04-01 19:08:51.405808640 +0200
@@ -83,7 +83,10 @@ int LYRA2(void *K, unsigned int kLen, co
 
     uint64_t i;                 //auxiliary iteration counter
     //==========================================================================/
-
+    if (nRows < 3)
+      return -1;
+    if (timeCost < 1)
+      return -1;
     //========== Initializing the Memory Matrix and pointers to it =============//
     //Tries to allocate enough space for the whole memory matrix
     i = (uint64_t) ((uint64_t)nRows * (uint64_t)ROW_LEN_BYTES);
@@ -288,7 +291,12 @@ int LYRA2(void *K, unsigned int kLen, co
     if (pKeys == NULL) {
         return -1;
     }
-    
+
+    if (nRows < 3)
+      return -1;
+    if (timeCost < 1)
+      return -1;
+
 #if _OPENMP <= 201107  //OpenMP 3.X or less 
     #pragma omp parallel num_threads(nPARALLEL) default(none) /*private(pwd)*/ shared(memMatrix,  pKeys, pwd, pwdlen, salt, saltlen, nRows, nCols, kLen, timeCost)
 #endif // _OPENMP
