Add library to makefile.
---
 makefile |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

Index: lyra2/makefile
===================================================================
--- lyra2.orig/makefile	2014-04-08 09:07:06.000000000 +0200
+++ lyra2/makefile	2014-09-20 09:44:40.576584654 +0200
@@ -8,7 +8,7 @@ CC?=gcc
 NVCC=nvcc
 CFLAGS=-std=c99 -Wall -pedantic -O3
 
-BINDIR=../bin
+BINDIR=bin
 BIN=$(BINDIR)/Lyra2
 BINCUDA=$(BINDIR)/Lyra2CUDA
 nCols=64
@@ -20,7 +20,7 @@ PTX = 10  # Must suits the GPU board use
 SM = 20   # Must suits the GPU board used.
 CUDADIR=cuda/
 
-default:
+default: generic-x86-64 lyra2.a
 	
 	@echo " "
 	@echo "To build Lyra2, type:"
@@ -39,6 +39,9 @@ default:
 	@echo " "
 
 
+lyra2.a: $(SSEDIR)Lyra2.o $(SSEDIR)Sponge.o
+	ar rcs $@ $^
+
 generic-x86-64:	    Lyra2.c Sponge.c Main.c Lyra2.h Sponge.h
 	mkdir -p $(BINDIR)	
 	$(CC) $(CFLAGS) Sponge.c Lyra2.c Main.c -o $(BIN) -DN_COLS=$(nCols) -O3
@@ -70,6 +73,6 @@ win-cygwin-x86-64-sse2:	$(SSEDIR)Lyra2.c
 	@echo "Build completed, binaries in $(BIN)"
 	
 clean:
-	rm -rf *.o $(BIN) $(BINCUDA)
+	rm -rf *.o *.a $(BIN) $(BINDIR) $(BINCUDA) $(SSEDIR)*.o
 
 
