---
 Makefile |   20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

Index: lanarea/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ lanarea/Makefile	2014-09-20 11:13:26.649142492 +0200
@@ -0,0 +1,20 @@
+CFLAGS=-O2 -Wall
+#LDLIBS=-lgcrypt -lssl -lnss3
+CC=gcc
+
+SOURCES=$(wildcard *.c)
+OBJECTS=$(SOURCES:.c=.o)
+
+all: lanarea.a
+
+lanarea.a: libb2/src/.libs/libb2_la-blake2b.o lanarea.o
+	ar rcs $@ $^
+
+libb2/src/.libs/libb2_la-blake2b.o:
+	(cd libb2; autoreconf; automake --add-missing; ./configure && make)
+
+clean:
+	rm -f *.o *~ core *.a
+	(cd libb2; make distclean)
+
+.PHONY: clean
