Add library Makefile.
---
 Makefile |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)

Index: argon2d/Makefile
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ argon2d/Makefile	2015-04-30 17:05:21.605471888 +0200
@@ -0,0 +1,17 @@
+CC=g++
+CPPFLAGS=-lpthread -std=c++11 -O2
+#LIBS=-pthread -lcrypto -lm
+
+TARGETS=argon2d.a
+
+SOURCES_ARGON2D=$(wildcard ref/*.cpp)
+
+default: $(TARGETS)
+
+argon2d.a: $(SOURCES_ARGON2D:.cpp=.o)
+	ar rcs $@ $^
+
+clean:
+	rm -f ref/*.o $(TARGETS)
+
+.PHONY: clean
