---
 Makefile |    7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

Index: parallel/Makefile
===================================================================
--- parallel.orig/Makefile	2014-09-20 11:27:09.861267605 +0200
+++ parallel/Makefile	2014-10-19 14:08:36.830683719 +0200
@@ -4,9 +4,12 @@ CFLAGS32 = -c -Wall -m32 -O2
 LFLAGS64 = -m64
 LFLAGS32 = -m32
  
-all: parallel-64 parallel-32
+#all: parallel-64 parallel-32
+all: parallel.a
 	
 
+parallel.a: parallel64.o sha51264.o
+	ar rcs $@ $^
 # 64 bit
 parallel-64: main64.o parallel64.o sha51264.o
 	$(CC) $(LFLAGS64) -o parallel-64 main64.o parallel64.o sha51264.o
@@ -34,4 +37,4 @@ sha51232.o: sha512.cpp sha512.h common.h
 	$(CC) $(CFLAGS32) -o sha51232.o sha512.cpp
 
 clean:
-	-rm *.o parallel-64 parallel-32
+	-rm -f *.o *.a parallel-64 parallel-32
