---
 battcrypt.h |    2 ++
 1 file changed, 2 insertions(+)

Index: battcrypt/battcrypt.h
===================================================================
--- battcrypt.orig/battcrypt.h	2015-04-28 12:32:06.827747573 +0200
+++ battcrypt/battcrypt.h	2015-04-28 12:32:06.823747565 +0200
@@ -5,6 +5,8 @@
 
 #include "common.h"
 
+extern "C" {
 int PHS(void *out, size_t outlen, const void *in, size_t inlen, const void *salt, size_t saltlen, unsigned int t_cost, unsigned int m_cost);
+}
 
 #endif
